deploy/charts/emqx/values.yaml

```diff
@@ -134,7 +134,7 @@ affinity: {}
 ingress:
   ## ingress for EMQX Dashboard
   dashboard:
-    enabled: false
+    enabled: true
```

Comment:
I think we should leave it at the default value of false, because not everyone will use the ingress